content/blog/cli-ai-extensions/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Pulumi Updates, Explained: AI-Powered Features in Pulumi CLI"
```

Comment:
Maybe just me, but this parses a little funny. Any reason not to just say

```suggestion
title: "AI-Powered Infrastructure as Code Features in Your CLI"
```

Reply:
The intent was to say that pulumi updates are now explained with AI, but maybe it came out a little awkward. 
Maybe simply "AI-Powered Features in Pulumi CLI"?  ("Infrastructure as Code" is applicable to everything we do and feels a bit redundant here)